cleverhans/attacks.py

```diff
@@ -13,18 +15,26 @@ class Attack(object):
 
     def __init__(self, model, back='tf', sess=None):
         """
-        :param model: A function that takes a symbolic input and returns the
-                      symbolic output for the model's predictions.
+        :param model: An instance of the cleverhans.Model class, or optionally
```

Comment:
I think it's OK to just call it `Model`. There actually isn't a `cleverhans.Model` at the moment, though I think it would be fine to add an import of the `Model` class at the base module